ts/Series/Sankey/SankeySeriesDefaults.ts

```diff
@@ -213,6 +213,27 @@ const SankeySeriesDefaults: PlotOptionsOf<SankeySeries> = {
      * @apioption plotOptions.sankey.levels.states
      */
 
+    /**
+     * Determines color mode for sankey links. Available options:
+     *
+     * - `from` color of the sankey link will be the same as the 'from node'
+     *
+     * - `gradient` color of the sankey link will be set to gradient between
+     * colors of 'from node' and 'to node'
+     *
+     * - `to` color of the sankey link will be same as the 'to node'.
+     *
+     * @sample highcharts/demo/vertical-sankey
+     *         Vertical sankey diagram with gradients
+     * @sample highcharts/series-sankey/link-color-mode
+     *         Sankey diagram with gradients and explanation
+     *
+     * @type      {('from'|'gradient'|'to')}
+     * @since     @next
+     * @apioption plotOptions.sankey.levels.linkColorMode
```

Comment:
The `linkColorMode` option is not only on `levels`. It is primarily a top-level option on the sankey series. This `@apioption` tag can probably be removed, since all the members on this object literal belong to the series options. As you can see with other options like `linkOpacity` and `opacity`. But I highly recommend you build the API (`gulp jsdoc`) and verify that the option is documented at the right place of the tree.

Reply:
Yes, I am currently pushing new changes, I did another oversight during the previous commit... I am really sorry for that. Totally understand, that this option is not on `levels`.